tests/test_service.py

```diff
@@ -288,3 +288,30 @@ def test_positive_foreman_maintain_service_list_sidekiq(ansible_module):
         assert "dynflow-sidekiq@orchestrator" in result["stdout"]
         assert "dynflow-sidekiq@worker" in result["stdout"]
         assert "dynflow-sidekiq@worker-hosts-queue" in result["stdout"]
+
+
+def test_positive_foreman_maintain_service_status_bz1945916(ansible_module, setup_rpmsave_file):
+    """Verify foreman-maintain service status does not get failed because of .rpmsave files
```

Comment:
```suggestion
    """Verify foreman-maintain service status does not fail because of .rpmsave file
```
Its not clear exactly why it fails. Do you mean "because of missing .rpmsave file" ?

Reply:
No, it was failing because of backup files like *.rpmsave

Reply:
Ok, so my suggestion stands.

Reply:
Updated.